docs/resources/identity_group_membership_v3.md

```diff
@@ -54,3 +54,11 @@ The following attributes are exported:
 * `group` - See Argument Reference above.
 
 * `users` - See Argument Reference above.
+
+## Import
+
+A group membership can be imported using the `id`, e.g.
```

Comment:
actually, users can not find the resource ID as it was generated by provider when creating.

I think we can delete the **Imported** in provider.